src/components/UserIcon/UserIcon.module.css

```diff
@@ -0,0 +1,23 @@
+.userIcon {
+  display: block;
+  width: 2.5rem;
+  height: 2.5rem;
```

Comment:
Add a variable for width and height? These should probably always be the same value